seungjun/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = YOUR PORT NUMBER SHOULD BE HERE!
+
+TYPEORM_CONNECTION = mysql
+TYPEORM_HOST = 127.0.0.1
+TYPEORM_USERNAME = root
+TYPEORM_PASSWORD = myPassword
+TYPEORM_DATABASE = myDB
+TYPEORM_PORT = 3306
+TYPEORM_LOGGING =TRUE
```

.env.sample은 말그대로 실제 환경변수를 관리하는 .env 파일이 어떤 변수명에 어떠한 값들을 담고있는지 나타내는 샘플 파일 입니다. 다른 변수들에 대해서도 전반적으로 실제 값이 아닌, 중의적인 표현으로 작성될 수 있게 수정해주세요!

```suggestion
PORT = YOUR PORT NUMBER SHOULD BE HERE!

TYPEORM_CONNECTION = rdbms
TYPEORM_HOST = db_host
TYPEORM_USERNAME = db_username
TYPEORM_PASSWORD = myPassword
TYPEORM_DATABASE = myDB
TYPEORM_PORT = db_port
TYPEORM_LOGGING =TRUE
```